src/main/java/nextstep/security/oauth2/GithubOauth2Client.java

```diff
@@ -0,0 +1,6 @@
+package nextstep.security.oauth2;
+
+public interface GithubOauth2Client extends OAuth2Client {
```

Comment:
이름을 통일해보면 좋을 것 같습니다.
```suggestion
public interface GithubOAuth2Client extends OAuth2Client {
```